EpikCord/managers/channel_manager.py

```diff
@@ -24,4 +24,11 @@ async def fetch(self, channel_id, *, skip_cache: Optional[bool] = False):
         channel = self.client.utils.channel_from_type(channel_data)
 
         self.channels.append(channel)
-        return channel
\ No newline at end of file
+        return channel
+
+    async def add(self,guild_id,data:dict):
+        channel_data = await self.client.http.post(f"guilds/{guild_id}/channels", data=data)
+        self.channels.append(channel_data)
+        
+    async def edit_pos(self,guild_id,data:dict):
```

Comment:
Use the entire name for the func, `edit_pos` isn't helpful and it's misleading and you should probably use Kwargs